### PR TITLE
Add Spanish (El Salvador) translations

### PR DESCRIPTION
**Pull request by roosemberth**
Added new translation file.

I have not built the project, so I'm not sure the contents of this patch series are enough to make it available. Feel free to comment on the MR or push to the branch shall there be something missing.

### CODE REVIEW

File: src/lib/translations/index.ts [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import { merge } from 'lodash-es';
 import en from './en.json';
+import es_sv from './es-sv.json';
 import fr from './fr.json';
 import { typedKeys } from '$lib/utils/typedKeys';
 
 export const languages = {
 	en,
+	es_sv,
```
Comment (coyotte508) [inline src/lib/translations/index.ts:9]:
```suggestion
	"es-sv": es_sv,
```

File: src/lib/translations/index.ts
Comment (coyotte508) [inline src/lib/translations/index.ts:1]: Please also add an entry for `"es-sv"` in `languageNames` :) 

(just added the variable to the codebase in https://github.com/B2Bitcoin/beBOP/commit/5f287dd666fcc176f8ca44cdba4792c507d7f571)

**Comment by coyotte508** [comment]
Looks good! Thank you for the contribution!

image

image

image

Maybe just the "cart.cta.view" text could be shorter

File: src/lib/translations/index.ts (hunk 1 above)
Comment (roosemberth) [reply src/lib/translations/index.ts]: Applied suggestion.

File: src/lib/translations/index.ts
Comment (roosemberth) [reply src/lib/translations/index.ts:1]: Applied suggestion.

**Comment by roosemberth** [comment]
> Maybe just the "cart.cta.view" text could be shorter (see first screenshot)

Rephrased a bit taking into account the context, but couldn't come up with a much shorter wording.

**Review by coyotte508 — APPROVED** [review]
Thank you for taking the time!

Will let @Tirodem check in before merging - might be a few days given the busy week he had :)

**Comment by Tirodem** [comment]
@coyotte508 Can I deploy it with deploy action as it's not carried by a proper be-BOP repo branch ?
before introducing it publicly, I think we'll need a new issue to manage which languages you want to enable or not, otherwise, some pilotes will have a "es-sv" option (will it appear as "es-sv" in language selector ?) despite not having any translation ready
it'll be better to schedule a "_🦋 In admin back-office, manage languages displayed in language selector_" feature in addition to this new (precious) ressource.

**Comment by coyotte508** [comment]
> @coyotte508 Can I deploy it with deploy action as it's not carried by a proper be-BOP repo branch ?

I don't think so, you'd have to copy the branch to a branch on the repo


>  before introducing it publicly, I think we'll need a new issue to manage which languages you want to enable or not, otherwise, some pilotes will have a "es-sv" option (will it appear as "es-sv" in language selector ?) despite not having any translation ready it'll be better to schedule a "_🦋 In admin back-office, manage languages displayed in language selector_" feature in addition to this new (precious) ressource. WDYT ?

Sure why not

**Comment by coyotte508** [comment]
Thank again @roosemberth !

**Comment by Tirodem** [comment]
@roosemberth next beer is for me 👌thanks for this contribution !
